src/ngx_http_lua_shdict.c

```diff
@@ -2843,6 +2843,38 @@ ngx_http_lua_ffi_shdict_flush_all(ngx_shm_zone_t *zone)
 
     return NGX_OK;
 }
+
+
+void
+ngx_http_lua_ffi_shdict_get_stats(ngx_shm_zone_t *zone,
+    size_t *total_used, size_t *total_size)
+{
+    ngx_http_lua_shdict_ctx_t   *ctx;
+    ngx_slab_stat_t             *stat;
+    size_t                       slot_count;
+    size_t                       slot_size;
+    size_t                       i;
+    size_t                       sum_used = 0;
+    size_t                       sum_size = 0;
+
+    ctx = zone->data;
+
+    ngx_shmtx_lock(&ctx->shpool->mutex);
+    slot_count = ngx_pagesize_shift - ctx->shpool->min_shift;
+    stat = ctx->shpool->stats;
+    slot_size = ctx->shpool->min_size;
+    for (i = 0; i < slot_count; i++) {
```

Comment:
Style: need a blank line before the for statement block.

Reply:
done